pkg/policy/repository.go

```diff
@@ -753,7 +753,7 @@ func (p *Repository) computePolicyEnforcementAndRules(securityIdentity *identity
 		if !ingress || !hasIngressDefaultDeny { // short-circuit len()
 			if len(r.Ingress) > 0 || len(r.IngressDeny) > 0 {
 				ingress = true
-				if *r.EnableDefaultDeny.Ingress {
+				if r.EnableDefaultDeny.Ingress == nil || *r.EnableDefaultDeny.Ingress {
```

Comment:
I'd suggest sanitizing the rules properly instead, then we shouldn't need extra nil checks much later in processing like this.
```suggestion
				if *r.EnableDefaultDeny.Ingress {
```

Reply:
done.